Assets/MixedRealityToolkit.Services/InputSimulation/InputSimulationService.cs

```diff
@@ -211,17 +211,13 @@ public override void Initialize()
         {
             base.Initialize();
 
-            ArticulatedHandPose.LoadGesturePoses();
```

Comment:
Since the JSON files are moving to classes, how will this impact ability to have custom input simulation gestures?

Reply:
hmmm, I mean the original implementation was hardcoded as well. 

But it is a good point to consider the flexibility of this new design. One could argue that the SimulatedArticulatedHandposes are a custom cache implementation and thus someone could create their own just like it...they would just have to update the other service implementations as well.

I could look at adding accessors to SimulatedArticulatedHandposes

Reply:
Added setter to override